packages/webapp/src/containers/Map/useDrawingManager.js

```diff
@@ -75,6 +77,9 @@ export default function useDrawingManager() {
         ...getDrawingOptions(type).polygonOptions,
       });
       redrawnPolygon.setMap(map);
+      maps.event.addListener(redrawnPolygon.getPath(), 'set_at', () => {
```

Comment:
I think this needs to be removed from the dom in the return statement of the useEffect.